src/engine/bank01.asm

```diff
@@ -183,36 +183,37 @@ MainDuelLoop: ; 40ee (1:40ee)
 	pop af
 	ldh [hWhoseTurn], a
 	call Func_3b21
+
+; animate the duel result screen
+; load the correct music and animation depending on result
 	ld a, [wDuelFinished]
 	cp TURN_PLAYER_WON
 	jr z, .active_duelist_won_battle
 	cp TURN_PLAYER_LOST
 	jr z, .active_duelist_lost_battle
-	ld a, $5f
+	ld a, DUEL_ANIM_DUEL_DRAW
 	ld c, MUSIC_DARK_DIDDLY
```

Comment:
Wow, I don't know how it escaped me this long, but I thought the thing I named "dark diddly" was unused.. I guess now it should be renamed `MUSIC_MATCH_DRAW`.